Userland/Utilities/cal.cpp

```diff
@@ -19,10 +19,9 @@ int const column_width = 22;
 char print_buffer[line_width * line_count];
 char temp_buffer[line_width * 8];
 
-int target_day;
-
-int current_year;
-int current_month;
+static int current_year;
+static int current_month;
+static int current_day;
```

Comment:
`static` seems redundant, there's no other TU for this application.

Reply:
Alright, removed.